pkg/plugins/runtime/gateway/route_table_generator.go

```diff
@@ -36,6 +37,16 @@ func GenerateVirtualHost(
 		)
 	}
 
+	if len(routes) == 0 {
+		routeBuilder := route.RouteBuilder{}
+
+		routeBuilder.Configure(route.RouteMatchExactPath("/"))
+		routeBuilder.Configure(route.RouteActionDirectResponse(http.StatusNotFound, "This is a Kuma MeshGateway! No route specified in your gateway"))
```

Comment:
```suggestion
		routeBuilder.Configure(route.RouteActionDirectResponse(http.StatusNotFound, "This is a Kuma MeshGateway. No routes match this MeshGateway!"))
```

And can we put this string constant outside this function?

Reply:
sure !!